lib/konbata.rb

```diff
@@ -49,18 +50,33 @@ def self.convert_courses
     course_structures.each_with_index do |(course_code, volumes), index|
       Konbata::Reporter.start_course(course_code, index, course_structures.size)
 
-      course = Konbata::Course.new(course_code, volumes)
+      course = Konbata::DocCourse.new(course_code, volumes)
       create_imscc(course)
 
       Konbata::Reporter.complete_course(course_code)
     end
   end
 
+  def self.convert_scorm
+    FileUtils.mkdir_p(OUTPUT_DIR)
+
+    scorm_package_paths = Dir.glob("#{INPUT_DIR}/*.zip")
+    scorm_package_paths.each do |package_path|
+      # Formats path to not have any spaces as the Scorm upload can't handle it
+      formatted_path = package_path.gsub(/[ ]/, "_")
```

Comment:
I think it'd be better to use `/\s/` for the `gsub` regex instead of `[ ]`. I feel like it's a little clearer.